Maven build fails

Build for accumulo-testing is failing with the following errors:

```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.8.1:compile (default-compile) on project accumulo-testing: Compilation failure: Compilation failure: 
[ERROR] /home/jeffrey/git/accumulo-testing/src/main/java/org/apache/accumulo/testing/randomwalk/Module.java:[227,89] incompatible types: boolean cannot be converted to java.util.concurrent.BlockingQueue<java.lang.Runnable>
[ERROR] /home/jeffrey/git/accumulo-testing/src/main/java/org/apache/accumulo/testing/randomwalk/bulk/Setup.java:[64,9] incompatible types: boolean cannot be converted to java.util.concurrent.BlockingQueue<java.lang.Runnable>
```
The current commit is 711d28e83a91cbaa0754064a7d72c0f975f7de31 though I have had successful builds on this commit. 
Command ran was maven clean package.

--- a/src/main/java/org/apache/accumulo/testing/TestProps.java
+++ b/src/main/java/org/apache/accumulo/testing/TestProps.java
@@ -94,7 +94,8 @@ public class TestProps {
   public static final String CI_INGEST_PAUSE_DURATION_MAX = CI_INGEST + "pause.duration.max";
   // Amount of data to write before flushing. Pause checks are only done after flush.
   public static final String CI_INGEST_FLUSH_ENTRIES = CI_INGEST + "entries.flush";
-  // The probability (between 0.0 and 1.0) that a set of entries will be deleted during continuous ingest
+  // The probability (between 0.0 and 1.0) that a set of entries will be deleted during continuous
+  // ingest
   public static final String CI_INGEST_DELETE_PROBABILITY = CI_INGEST + "delete.probability";
 
   /** Batch Walker **/

--- a/src/main/java/org/apache/accumulo/testing/randomwalk/Module.java
+++ b/src/main/java/org/apache/accumulo/testing/randomwalk/Module.java
@@ -224,7 +224,7 @@ public class Module extends Node {
       fixture.setUp(state, env);
     }
 
-    ExecutorService service = ThreadPools.createFixedThreadPool(1, "RandomWalk Runner", false);
+    ExecutorService service = ThreadPools.createFixedThreadPool(1, "RandomWalk Runner");
 
     try {
       Node initNode = getNode(initNodeId);

--- a/src/main/java/org/apache/accumulo/testing/randomwalk/bulk/Setup.java
+++ b/src/main/java/org/apache/accumulo/testing/randomwalk/bulk/Setup.java
@@ -60,8 +60,7 @@ public class Setup extends Test {
     state.set("fs", FileSystem.get(env.getHadoopConfiguration()));
     state.set("bulkImportSuccess", "true");
     BulkPlusOne.counter.set(0l);
-    ThreadPoolExecutor e = ThreadPools.createFixedThreadPool(MAX_POOL_SIZE, "bulkImportPool",
-        false);
+    ThreadPoolExecutor e = ThreadPools.createFixedThreadPool(MAX_POOL_SIZE, "bulkImportPool");
     state.set("pool", e);
   }
 

